tests/integration/conftest.py

```diff
@@ -109,14 +106,14 @@
 
 
 @pytest.fixture(scope="session")
-def dummy_census_data(tmp_path_factory):
+def decennial_census_data_path(tmp_path_factory):
     """Generate a dummy decennial census dataframe, save to a tmpdir, and return that path."""
     random.seed(0)
     num_rows = 100_000
     data = pd.DataFrame(
         {
             "housing_type": [random.choice(HOUSING_TYPES) for _ in range(num_rows)],
-            "age": [str(random.random() * 100) for _ in range(num_rows)],
+            "age": [str(random.randint(1, 100)) for _ in range(num_rows)],
```

Comment:
A followup PR will address this - post-processing should change all of the observed ages (which are floats) to floors, ie no one would type their age as "29.99999" but would instead use "29".